IndicoV2.Tests/Extensions/SubmissionResult/SubmissionResultAwaiterTests.cs

```diff
@@ -20,7 +20,7 @@ namespace IndicoV2.Tests.Extensions.SubmissionResult
     public class SubmissionResultAwaiterTests
     {
         private static readonly SubmissionStatus[] _submissionStatusesExceptProcessingAndFailed =
-            Enum.GetValues(typeof(SubmissionStatus)).Cast<SubmissionStatus>().Where(s => s != SubmissionStatus.PROCESSING && s != SubmissionStatus.FAILED && s != SubmissionStatus.POST_PROCESSING).ToArray();
+            Enum.GetValues(typeof(SubmissionStatus)).Cast<SubmissionStatus>().Where(s => s is not SubmissionStatus.PROCESSING and not SubmissionStatus.FAILED).ToArray();
```

Comment:
Visual Studio changed `!=` to `is not` --  happy to switch it back if we want

Reply:
Let's keep it how it is for now/revert this so there's no code changes.

Reply:
cool, changed it back